src/mainboard/clevo/tgl-u/variants/ns50mu/hda_verb.c

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
any differences between nv41 and this board?

Reply:
Yes, the subsystem ID and one line of pin config are different